src/service/server/QueryHandler.cpp

```diff
@@ -330,7 +332,8 @@ std::shared_ptr<Expression> QueryHandler::buildMetric(const Metric& metric) cons
     return std::make_shared<decltype(exp)>(exp);                                             \
   }
 
-// TODO(shuo): get min/max for each column and pass in to hist udaf
+  // TODO(shuo): get min/max for each column and pass in to hist udaf
+  // now we can get given column stats by this call BlockManager::init()->hist
```

Comment:
@shuoshang1990 this is what you need, we may want to make hist method to accept column name, right now it expect you find the column index and pass index in (check method `buildPredicate` in current file to find how to get column index for name).

Reply:
correct, you will need to get column index and call `BlockManager::init()->hist("table name", "column index")` to get the histogram object, and read min/max out of it.